src/object.d

```diff
@@ -3190,11 +3199,24 @@ void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
  * Create RTInfo for type T
  */
 
+template RTInfoImpl(size_t[] pointers)
+{
+    immutable size_t[pointers.length] data = pointers[];
+    immutable RTInfoImpl = data.ptr;
```

Comment:
> 16 bytes pointer bitmap already covers 1024 bytes of memory (assuming 64-bit pointers), so it won't fit into the same cache line for most blocks.

Right, that won't cut it. They only vague idea I have until now is to group allocations with the same pointer bitmaps. That could make sense in general (grouping similar objects) and would get us a bit towards [Connectivity-Based Garbage Collection](https://www.cs.purdue.edu/homes/hosking/690M/cbgc.pdf).
There are only 2^(16/8)=4 binary combinations for 16-byte sized objects, 2^(64/8)=256 for 64-byte objects. Not all of them will get used, so we could try to keep them on separate pages and freelists. This would be even more interesting if we could allocate arrays as multiple small size allocations, but for now we could just use separate pools for them as well.
For objects bigger than 64-byte we can use GCBits or try to store the bits with the objects.